### ISSUE TITLE
[BUG] 部分页面上无法拖动ScrollBar

### ISSUE THREAD

**Issue by kavinzhao** [DanXi-Dev/DanXi#51]
错误`Scroll Controller attached to multiple views`

由于必须使用`PrimaryScrollController`，需要另寻他法解决此问题

**Comment by kavinzhao** [comment]
https://www.jianshu.com/p/ab473fb8ceb0 Perhaps this might help?

**Comment by w568w** [comment]
Can we designate the `PrimaryScrollController` to be new controller created by ourselves?

**Comment by kavinzhao** [comment]
> Can we designate the `PrimaryScrollController` to be new controller created by ourselves?

No. PrimaryScrollController is an inherited widget.

However, we may try to detach some of the views from the controller.

We can query the views via `controller.positions`, which returns a list of all positions attached to the controller, and use `controller.detach()` to detach a position.

**Comment by w568w** [comment]
> No. PrimaryScrollController is an inherited widget.

Then FUCK flutter, not until its community finds how stupid the design of `PrimaryScrollController` and the fucking state management of tab pages and `ScrollView` on them are and trys to have a solution, will we spend any more precious time on this annoying problem. State's lifecycle callbacks are just as poor as nothing.

**Comment by kavinzhao** [comment]
Well, looks like it’s mostly fixed. Although there are errors being thrown from time to time under the current implementation, considering the fix is functional, I’m going to close the issue for now.
